preferences/Preference.js

```diff
@@ -1,19 +1,23 @@
 // See https://github.com/jenkinsci/js-storage
-const storage = require('@jenkins-cd/storage');
-const jenkinsNS = storage.jenkinsNamespace();
-const preferences = jenkinsNS.subspace('preferences');
+const storage = require('@jenkins-cd/storage/storage');
+const StorageNamespace = require('@jenkins-cd/storage/StorageNamespace');
```

Comment:
Better to not include things this way, can cause issues/duplication in the bundler.